File: app/src/main/java/com/absinthe/kage/device/heartbeat/HeartbeatRequest.java
package com.absinthe.kage.device.heartbeat;

import com.absinthe.kage.connect.tcp.Request;
import com.absinthe.kage.device.cmd.HeartbeatCommand;

public class HeartbeatRequest extends Request {

    public HeartbeatRequest() {
        setData(new HeartbeatCommand().pack());
    }
}

File: app/src/main/java/com/absinthe/kage/device/heartbeat/HeartbeatRequest.kt
package com.absinthe.kage.device.heartbeat

import com.absinthe.kage.connect.tcp.Request
import com.absinthe.kage.device.cmd.HeartbeatCommand

class HeartbeatRequest : Request() {

    init {
        data = HeartbeatCommand().pack()
    }

}